src/fundus/publishers/us/__init__.py

```diff
@@ -164,3 +165,13 @@ class US(PublisherEnum):
         ],
         parser=BusinessInsiderParser,
     )
+
+    RollingStone = PublisherSpec(
+        name="Rolling Stone",
+        domain="https://www.rollingstone.com/",
+        sources=[
+            NewsMap("https://www.rollingstone.com/news-sitemap.xml"),
+            Sitemap("https://www.rollingstone.com/sitemap_index.xml"),
```

Comment:
There is some bloat in the sitemap Fundus won't be able to parse. Could you update the sitemap like this:

```python
Sitemap(
    "https://www.rollingstone.com/sitemap_index.xml",
    sitemap_filter=inverse(lor(regex_filter("post-sitemap"), regex_filter("pmc_list-sitemap"))),
)
```

You can import everything from `fundus.scraping.filter`:

```python
from fundus.scraping.filter import regex_filter, inverse, lor
```